Makefile.core.mk

```diff
@@ -365,7 +366,21 @@ build-linux: depend
 	STATIC=0 GOOS=linux GOARCH=amd64 LDFLAGS='-extldflags -static -s -w' common/scripts/gobuild.sh $(ISTIO_OUT_LINUX)/ $(BINARIES)
 
 # Create targets for ISTIO_OUT_LINUX/binary
-$(foreach bin,$(BINARIES),$(ISTIO_OUT_LINUX)/$(shell basename $(bin))): build-linux
+# There are two use cases here:
+# * Building all docker images (generally in CI). In this case we want to build everything at once, so they share work
+# * Building a single docker image (generally during dev). In this case we just want to build the single binary alone
```

Comment:
This is not strictly necessary, but is another optimization made possible by this change. This way both CI and dev experience is optimized